comfy_cli/command/distribution.py

```
@@ -841,31 +1077,52 @@ def create_cmd(
-            f"({git_nodes} from git, {len(plan['definition']['customNodes']) - git_nodes} to upload)"
+            f"  custom nodes: {len(nodes_out)}  "
+            f"({git_nodes} from git, {registry_nodes} from the registry, {node_uploads} to upload)"
         )
         renderer.print("\n  Builder definition that would be sent:")
         renderer.console().print_json(json.dumps(plan["definition"]))
 
     renderer.emit({"name": name, "plan": plan, "executed": False}, command="distribution create")
 
 
-def _create_execute(renderer, definition: dict, *, name: str, builder_url: str | None, models_dir: str | None) -> None:
+def _create_execute(
```

Pylint (4.0.6)

[refactor] Too many arguments (6/5)

(R0913)